src/extend-config/extendConfig.ts

```diff
@@ -1,32 +1,49 @@
 import { FabloConfigJson } from "../types/FabloConfigJson";
-import { FabloConfigExtended } from "../types/FabloConfigExtended";
-import { extendRootOrgConfig, extendOrdererOrgsConfig, extendOrgsConfig } from "./extendOrgsConfig";
+import { FabloConfigExtended, OrdererGroup, OrgConfig } from "../types/FabloConfigExtended";
+import { extendOrgsConfig } from "./extendOrgsConfig";
 import extendNetworkSettings from "./extendNetworkSettings";
 import extendChannelsConfig from "./extendChannelsConfig";
 import extendChaincodesConfig from "./extendChaincodesConfig";
+import _ = require("lodash");
+
+const mergeOrdererGroupsByName = (orgs: OrgConfig[]): OrdererGroup[] => {
+  const ordererGroups = orgs.flatMap((o) => o.ordererGroups);
+
+  const ordererGroupsGrouped: Record<string, OrdererGroup[]> = _.groupBy(ordererGroups, (group) => group.name);
+
+  return Object.values(ordererGroupsGrouped).flatMap((groupsWithSameGroupName) => {
+    const orderers = groupsWithSameGroupName.flatMap((group) => group.orderers);
+    const hostingOrgs = groupsWithSameGroupName.flatMap((group) => group.hostingOrgs);
+    const ordererHeads = groupsWithSameGroupName.flatMap((group) => group.ordererHeads);
```

Comment:
I'm not sure about the naming. `groupParts` instead of `groupsWithSameGroupName`?